.chloggen/splunkent-add-codeowner.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
no need for a changelog for this, please remove and i'll add a label to skip changelog checks 👍🏻

Reply:
heard! wasn't sure figured better safe than sorry.